lib/ansible/modules/packaging/os/satellite_subscription.py

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Luc Stroobant (luc.stroobant@wdc.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+
+DOCUMENTATION = """
+---
+module: satellite_subscription
+short_description: Red Hat Satellite subcription (license) mangement module
+description:
+  - This module allows to assign or remove subscriptions to hosts in the satellite
+  - It works on the Satellite side, so not from the client side like redhat_subscription.
+    Your hosts or hypervisors should be in the Satellite already. The module will run subscription
+    related tasks (assign, remove, autoattach or replace with hypervisor subscriptions) for the
+    hosts through the Satellite API.
+version_added: "2.8"
+requirements:
+  - requests
+author:
+  - Luc Stroobant (@stroobl)
+options:
+  satellite_hostname:
+    description:
+      - Red Hat Satelite server (6.x) hostname
```

Comment:
Typo. Always trailing dots in descriptions (except short_description). But we prefer a brief description of the parameter.
Also we have standardized parameter names for this, either `hostname` or `login_hostname`.

Just like we have `user` or `login_user` and `password` or `login_password`.
```suggestion
      - The name or IP address of the Red Hat Satellite server.
```
Do the same for the other descriptions.